app/scripts/inpage.js

```diff
@@ -98,4 +98,4 @@ setupDappAutoReload(web3, inpageProvider._publicConfigStore)
 // end deprecate:Q1-2020
 //
 
-window.ethereum = proxiedInpageProvider
+setGlobalProvider(proxiedInpageProvider)
```

Comment:
Is there some reason `initProvider` wasn't used here? I had thought you intended to use that to replace much of what's in this file.

Reply:
I hesitated because we aren't using the proxied provider to initialize `web3` in `setupDappAutoReload`, and because `web3` will now be set on `window` after `ethereum` is as opposed to before, but I just pushed a change trying it out.